scratch-gui/src/addons/api.js

```
@@ -743,7 +743,7 @@ class AddonRunner {
         this.resources = null;
 
         this.publicAPI = {
-            global,
+            global: globalThis,
```

ESLint

[error] 'globalThis' is not defined.

(no-undef)